server/controllers/refer/courseParticipationsController.ts

```diff
@@ -48,6 +48,29 @@ export default class CourseParticipationsController {
     }
   }
 
+  details(): TypedRequestHandler<Request, Response> {
+    return async (req, res) => {
+      TypeUtils.assertHasUser(req)
+      const { courseParticipationId, referralId } = req.params
+
+      await this.courseService.getParticipation(req.user.token, courseParticipationId)
```

Comment:
I don't think this needs to be here (at least in this commit)? Guessing it'll be used to pre-fill values later